strongbox-parent/pom.xml

```diff
@@ -68,13 +68,13 @@
         <version.httpclient>4.5</version.httpclient>
         <version.jackson>2.8.7</version.jackson>
         <version.javaee>7.0</version.javaee>
-        <version.jcl>1.7.24</version.jcl>
+        <version.jcl>1.8.0-beta0</version.jcl>
```

Comment:
Could this be set to the latest known official (non-beta) release, please?

Reply:
Yes, it is.